server/sonar-server/src/main/java/org/sonar/server/computation/measure/api/MeasureComputerImpl.java

```diff
@@ -21,10 +21,13 @@
 package org.sonar.server.computation.measure.api;
 
 import com.google.common.collect.ImmutableSet;
+import java.util.ArrayList;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'java.util.ArrayList'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)